modules/neo4j/config.go

```diff
@@ -3,19 +3,10 @@ package neo4j
 import (
```

Comment:
@fbiville please take a look at these changes, as the NeoJ module will be refactored after this PR